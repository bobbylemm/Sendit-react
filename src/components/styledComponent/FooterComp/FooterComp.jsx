import styled from 'styled-components';

const FooterComponent = styled.footer`
  background: #312c32;
  display: flex;
  justify-content: space-around;
  flex-flow: row wrap;
  padding: 3rem 6rem;
  font-weight: 600;
  color: #fff;

  span a {
    color: rgb(173, 166, 166);
  }
`;

export default FooterComponent;
