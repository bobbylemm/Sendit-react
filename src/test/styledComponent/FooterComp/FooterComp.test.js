import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FooterComp from '../../../components/styledComponent/FooterComp/FooterComp';

describe('testing the button element', () => {
  it('should mount button', () => {
    const tree = renderer.create(<FooterComp />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toHaveStyleRule('background', '#312c32');
  });
});
