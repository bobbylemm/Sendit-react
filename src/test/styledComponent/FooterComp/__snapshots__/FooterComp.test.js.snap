// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the button element should mount button 1`] = `
.c0 {
  background: #312c32;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  padding: 3rem 6rem;
  font-weight: 600;
  color: #fff;
}

.c0 span a {
  color: rgb(173,166,166);
}

<footer
  className="c0"
/>
`;
